dockerfiles/Dockerfile.webpack

```diff
@@ -2,4 +2,4 @@ FROM node:22.11
 COPY package-lock.json /usr/src/app/checkouts/ext-theme/
 COPY package.json /usr/src/app/checkouts/ext-theme/
 WORKDIR /usr/src/app/checkouts/ext-theme
-RUN npm ci
+RUN npm install
```

Comment:
As long as we are checking in built assets to the `ext-theme` repository, this command should be `npm ci`.

`npm install` will upgrade previously installed packages without warning.